src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,126 @@
+- is secondary, which means it is not likely that users clicks it in that context
+
+<limel-example-action-buttons-placement></limel-example-action-buttons-placement>
+
+:::tip
+In [`limel-dialog`](#/components/limel-dialog), there is a `slot` called `button`, which is flex-box. It automatically aligns its child elements to the right side. All you need to do is to write `slot="button"` on your elements.
+
+```tsx
+<limel-button
+    label="Ok"
+    slot="button"
+/>
+```
+:::
+
+<br/>
+
+*****
+
+## Primary versus secondary actions
+
+As you know, `limel-button` can have a `primary={true}` property, which makes it stand out by getting the primary color of your theme.
+
+One of the action buttons is normally the primary action. That is the action that you expect the users to take, or you want or prefer them to click on.
+
+<limel-example-action-buttons-primary-secondary></limel-example-action-buttons-primary-secondary>
+
+A primary button has a stronger visual weight, and appearing more distinct makes it easier for users to chose the correct, safe, or promoted action.
+
+You can use this visual distinction to promote the action that is safer, even if that action may not be the primary action. You can do so because sometimes:
+- it can be hard to identify the intention of the user
+- the action may be irreversible or risky
+
+<limel-example-action-buttons-primary-secondary-reversed></limel-example-action-buttons-primary-secondary-reversed>
+
+<br/>
+
+*****
+
+## Choosing good labels
+
+Labels you use should make sense together. You are usually dealing with contradictory actions. Therefore, what you put on each button should also reflect the contradiction.
+
+<limel-example-action-buttons-choosing-labels></limel-example-action-buttons-choosing-labels>
+
+Labels must be making each option as clear as possible. An explicit label give users more confidence in selecting the correct action.
+
+:::note
+- Make sure the label explains what the button does, rather than a generic label (such as “OK”, or "Yes").
+- Using verbs as labels instead of nouns is better, because a verb describes the action better.
+:::
+
+<limel-example-action-buttons-choosing-explicit-labels></limel-example-action-buttons-choosing-explicit-labels>
+
+<br/>
+
+*****
+
+## Providing a third alternative action
+
+Sometimes having two action is not enough for the user to take a decision with peace of mind. A common scenario is when a user has unsaved changes and tries to navigate away from the view. In such cases, typically a prompt maybe be shown to alert the user and give them a few logical choices.
+
+This prompt can offer two main actions of "Save" and "Discard" (which means "save & continue navigating away from this page" and "discard & continue navigating away from this page"), along with a third choice that allows users to "Cancel" (and go back / not continue navigating away!).
```

Comment:
```suggestion
This prompt can offer two main actions of "Save" and "Discard" (which means "save & continue navigating away from this page" and "discard & continue navigating away from this page"), along with a third choice that allows users to cancel (and go back / not continue navigating away!).
```

The quote marks and uppercase 'C' suggests "Cancel" as the actual label, like "Save" and "Discard", but "Cancel" isn't a good label for all the reasons outlined in the previous section 😄

Reply:
right 😄